<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1674</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1674</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecentlyUsedFiles">
     <property name="title">
      <string>Recently used files</string>
     </property>
     <addaction name="actionRecentFile1"/>
    </widget>
    <addaction name="actionSaveSession"/>
    <addaction name="actionRestoreSession"/>
    <addaction name="menuRecentlyUsedFiles"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuShow_View">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Show View</string>
     </property>
    </widget>
    <addaction name="menuShow_View"/>
    <addaction name="actionRestoreWindowPlacement"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuPlugins">
     <property name="title">
      <string>Plugins</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/images/plugin.png</normaloff>:/icons/images/plugin.png</iconset>
     </property>
    </widget>
    <addaction name="menuPlugins"/>
    <addaction name="actionSavePlugins"/>
    <addaction name="actionLoadPlugins"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="Main">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="logDockWidget">
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="filterSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LogView" name="logView">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionRestoreWindowPlacement">
   <property name="text">
    <string>Restore initial window placement</string>
   </property>
  </action>
  <action name="actionRecentFile1">
   <property name="text">
    <string>empty</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSaveSession">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/save-as.png</normaloff>:/icons/images/save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save Session ...</string>
   </property>
   <property name="toolTip">
    <string>Save Session information to xml file</string>
   </property>
  </action>
  <action name="actionRestoreSession">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/session-open.png</normaloff>:/icons/images/session-open.png</iconset>
   </property>
   <property name="text">
    <string>Restore Session ...</string>
   </property>
   <property name="toolTip">
    <string>Restore Session information from xml file</string>
   </property>
  </action>
  <action name="actionSavePlugins">
   <property name="text">
    <string>Save plugin configuration</string>
   </property>
  </action>
  <action name="actionLoadPlugins">
   <property name="text">
    <string>Load plugin configuration</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/images/configure.png</normaloff>:/icons/images/configure.png</iconset>
   </property>
   <property name="text">
    <string>Configure ricodebug</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogView</class>
   <extends>QTableView</extends>
   <header>.logview</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
